config/boards/khadas-edge2.conf

```diff
@@ -25,10 +25,17 @@ function post_family_tweaks__kedge2_naming_audios() {
 	return 0
 }
 
+declare -g DDR_BLOB='rk35/rk3588_ddr_lp4_2112MHz_lp5_2400MHz_v1.16.bin'
+declare -g BL31_BLOB='rk35/rk3588_bl31_v1.45.elf'
+declare -g BL32_BLOB='rk35/rk3588_bl32_v1.15.bin'
+
 # for the kedge2, we're counting on the blobs+u-boot in SPI working, as it comes from factory. It does not support bootscripts.
 function post_family_config__uboot_kedge2() {
-	display_alert "$BOARD" "Configuring ($BOARD) non-u-boot" "info"
-	unset BOOTSOURCE
-	declare -g BOOTCONFIG='none'
+	display_alert "$BOARD" "Configuring ($BOARD) u-boot" "info"
+
+	declare -g BOOTSOURCE='https://github.com/khadas/u-boot.git'
+	declare -g BOOTBRANCH='branch:khadas-edges-v2017.09'
+	declare -g BOOTPATCHDIR="legacy/u-boot-khadas-edge2-rk3588"
+	declare -g BOOTCONFIG="khadas-edge2-rk3588s_defconfig"
 	declare -g SRC_EXTLINUX="yes" # For now, use extlinux. Thanks Monka
```

Comment:
Good stuff. I suppose we can remove the `SRC_EXTLINUX` here and use boot.cmd/scr like other boards? kedge2 was different in this aspect; using boot.scr enables armbianEnv.txt, overlay selection, etc.